modules/extension/complex/src/main/java/org/geotools/data/complex/ComplexFeatureConstants.java

```diff
@@ -52,6 +52,16 @@
     public static final PropertyDescriptor FEATURE_CHAINING_LINK = new AttributeDescriptorImpl(
             XSSchema.STRING_TYPE, FEATURE_CHAINING_LINK_NAME, 0, -1, true, null);
 
+    /**
+     * Static attribute name used for the auxiliary default geometry attribute, which is created when
+     * &lt;defaultGeometry&gt; is specified in the feature type mapping configuration.
+     * 
+     * The constant represents to the local part of the qualified attribute name; the namespace URI
+     * is generated at runtime and will be the same as the namespace URI of the feature containing
+     * the geometry attribute.
+     */
+    public static final String DEFAULT_GEOMETRY_LOCAL_NAME = "__DEFAULT_GEOMETRY__";
```

Comment:
In the rest of GeoTools the default goemetry is represented by the empty string, "", was there a specific reason to use a different value?

Reply:
I was just no aware of that :yum:, updating the code to sue the empty String.